core-wih/src/main/java/io/elimu/a2d2/corewih/MixPanelWorkItemHandler.java

```diff
@@ -65,6 +65,32 @@ public void executeWorkItem(WorkItem workItem, WorkItemManager manager) {
 		}
 	}
 
+	private String prettify(String headerName) {
+		String startPoint = headerName.replace(MIXPANEL_REQUEST_PREFIX, "");
+		startPoint = startPoint.replaceAll("-", " ");
+		String[] split = startPoint.split(" ");
+		StringBuilder retval = new StringBuilder();
+		for (int index = 0; index < split.length; index++) {
+			String part = split[index];
+			if (part.length() < 1) {
+				continue;
+			}
+			if (index > 0) {
+				retval.append(' ');
+			}
+			retval.append(part.toUpperCase().charAt(0)).append(part.substring(1));
+		}
+		return retval.toString();
+	}
+
+	/*public static void main(String[] args) {
+		System.out.println(new MixPanelWorkItemHandler().prettify("mixpanel-organization-id"));
```

Comment:
@marianbuenosayres can we remove this commented code, and add unit test cases for all of these scenario?